ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -339,7 +340,7 @@ describe("Indexer", () => {
     it(
       "gets user transactions",
       async () => {
-        const userTransactions = await indexerClient.getUserTransactions(482294669, { limit: 4 });
+        const userTransactions = await indexerClient.getUserTransactions(undefined, { options: { limit: 4 } });
```

Comment:
Also in general, ideally any optional argument in TS isn't positional, it feels a bit weird to have to pass in `undefined` rather than just not setting it.

Reply:
start_version - can be set to tell indexer what version to start from
limit - specifies the maximum number of items or records to return in a query result.
offset - specifies the starting position of the query result within the set of data.

maybe this query interface makes more sense?
```
async getUserTransactions(extraArgs: {
    startVersion?: number;
    options?: IndexerPaginationArgs;
    orderBy?: IndexerSortBy<User_Transactions_Order_By>[];
  }
```

Reply:
Okay actually I was looking at the wrong function but my point stands:
```
async getUserTransactions(
    startVersion?: number,
    extraArgs?: {
      options?: IndexerPaginationArgs;
      orderBy?: IndexerSortBy<User_Transactions_Order_By>[];
    },
  ): Promise<GetUserTransactionsQuery> {
    const whereCondition: any = {
      version: { _lte: startVersion },
    };
```

If `start_version` is optional why not add it to somewhere in `extraArgs`?

Reply:
agreed! this might make more sense

```
async getUserTransactions(extraArgs?: {
    startVersion?: number;
    options?: IndexerPaginationArgs;
    orderBy?: IndexerSortBy<User_Transactions_Order_By>[];
  }
```

Reply:
done!